scripts/prepare_regions_data/prepare-regions-data.ts

```diff
@@ -39,6 +45,48 @@ async function main() {
   }
 }
 
+/**
+ * We optimize storing the URL segments, which can be computed from names.
```

Comment:
This first sentence doesn't quite make sense to me without more context.  Maybe something like:
```
/** 
 * Our location URLs must be consistent over time to avoid broken URLs, but to
 * save space we don't want to persist them in the frontend code.  So this
 * function validates that the computation functions used in the frontend code
 * generate url segments that exactly match the url segments stored in the raw
 * region data files.
 */
```